lab-matt/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const greet = require(__dirname + '/lib/howdy');
```

Comment:
i usualy name the variables that reference my modules after the name of their file,,, that way when the code is complex its easy to tell where things come from